src/test/java/seedu/guest/logic/parser/EditCommandParserTest.java

```diff
@@ -102,19 +109,21 @@ public void parse_invalidValue_failure() {
         assertParseFailure(parser, "1" + PHONE_DESC_BOB + INVALID_PHONE_DESC, Phone.MESSAGE_CONSTRAINTS);
 
         // multiple invalid values, but only the first invalid value is captured
-        assertParseFailure(parser, "1" + INVALID_NAME_DESC + INVALID_EMAIL_DESC + VALID_DATE_RANGE_AMY
+        assertParseFailure(parser, "1" + INVALID_NAME_DESC + INVALID_EMAIL_DESC
+                + VALID_ROOM_AMY + VALID_DATE_RANGE_AMY
                         + VALID_PHONE_AMY, Name.MESSAGE_CONSTRAINTS);
```

Comment:
Should the indentation of this line be the same as the line above it?

Reply:
Rectified!